File: JS Exams/01.  Basics Exam Collection/Basics Online Exam - 20 and 21 April 2019/02. Easter Guests/02. Easter Guests.js
function easterGuests(guestsCount, budget) {
    
    let oneCakePrice = 4
    let oneEggPrice = 0.45
    let allCakes = Math.ceil(guestsCount / 3)
    let allEggs = guestsCount * 2
    let cakesPrice = allCakes * oneCakePrice
    let eqq_price = allEggs * oneEggPrice
    let totalPrice = cakesPrice + eqq_price

    if (totalPrice <= budget)
        return `Lyubo bought ${allCakes} Easter bread and ${allEggs} eggs.\nHe has ${(budget - totalPrice).toFixed(2)} lv. left.`
    return `Lyubo doesn\'t have enough money.\nHe needs ${(totalPrice - budget).toFixed(2)} lv. more.`
}

//console.log(easterGuests(10, 35))

File: Python Exams/01. Basic Exam Collection/Basics Online Exam - 20 and 21 April 2019/02. Easter Guests/02. Easter Guests.py
from math import ceil

guests_count = int(input())
budget = int(input())

one_cake_price = 4
one_egg_price = 0.45

all_cakes = ceil(guests_count / 3)
all_eggs = guests_count * 2

cakes_price = all_cakes * one_cake_price
eqq_price = all_eggs * one_egg_price
total_price = cakes_price + eqq_price

if total_price <= budget:
    print(f'Lyubo bought {all_cakes} Easter bread and {all_eggs} eggs.\nHe has {budget - total_price:.2f} lv. left.')
else:
    print(f'Lyubo doesn\'t have enough money.\nHe needs {total_price - budget:.2f} lv. more.')
